README.md

```diff
@@ -24,7 +24,7 @@ ytdl('http://www.youtube.com/watch?v=A02s8omM_hI')
 
 Attempts to download a video from the given url. Returns a readable stream. `options` can have the following keys
 
-* `quality` - Video quality to download. Can be an [itag value](http://en.wikipedia.org/wiki/YouTube#Quality_and_formats) value, a list of itag values, or `highest`/`lowest`. Defaults to `highest`.
+* `quality` - Video quality to download. Can be an [itag value](http://en.wikipedia.org/wiki/YouTube#Quality_and_formats) value, a list of itag values, or `highest`/`lowest`/`highestaudio`/`highestvideo`. `highestaudio`/`highestvideo` both prefer only responding with only audio/video. Defaults to `highest`.
```

Comment:
>`highestaudio`/`highestvideo` both prefer only responding with only audio/video

Not necessarily. These two are meant to return the quality with the highest audio/video regardless of the quality of the other.

If it were the case that they'd respond with only audio/video, then we could just use `quality: 'highest, filter: 'audio/videoonly'`